pom.xml

```diff
@@ -25,57 +25,14 @@
     <version>8.4-12</version>
   </parent>
   <groupId>com.xwiki.googleapps</groupId>
-  <artifactId>application-googleapps</artifactId>
+  <artifactId>application-googleapps-parent</artifactId>
   <version>2.5-SNAPSHOT</version>
-  <packaging>xar</packaging>
-  <name>Google Apps Integration</name>
+  <packaging>pom</packaging>
+  <name>Google Apps Integration (pro) parent POM</name>
```

Comment:
Other extensions use `(Pro)` in this description. We should look for consistency.